website/content/en/docs/deployment/_index.md

```diff
@@ -3,3 +3,12 @@ title = "Deployment"
 description = "Deploy Kubeflow on AWS"
 weight = 10
 +++
+Kubeflow on AWS is differentiated from the open-source KubeFlow project by a custom build manifest that provides integrations with various AWS services that are highly available and scalable. This reduces the operational overhead of maintaining the Kubeflow platform. 
+
+If you want to deploy Kubeflow with minimal changes, but optimized for [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/) (Amazon EKS), then consider the [vanilla](/kubeflow-manifests/docs/deployment/vanilla/guide/) deployment option. The Kubeflow on AWS control plane is installed on top of Amazon EKS, which is a managed container service used to run and scale Kubernetes applications in the cloud.
+
+Kubeflow on AWS provides integration with the [Amazon Relational Database Service](https://aws.amazon.com/rds/) (RDS) for highly scalable and available pipelines and metadata store. RDS removes the need to manage a local MYSQL database service and storage. Kubeflow on AWS also provides integration with [Amazon Simple Storage Service](https://aws.amazon.com/s3/) (S3) for an easy-to-use pipeline artifacts store. S3 removes the need to host the local object storage MinIO. You can deploy Kubeflow on AWS with both [RDS and S3](/kubeflow-manifests/docs/deployment/rds-s3/guide/) or [using only RDS or only S3](/kubeflow-manifests/docs/deployment/rds-s3/guide/#using-only-rds-or-only-s3).
```

Comment:
Doesnt feel like we are telling the users to choose an option here. We want to convey that. I think we need to find a way to connect the content below with the first line.

also this content should have call to action